### PR TITLE
[Golang][Doc] improve the samples and doc

### PR DESCRIPTION
**Pull request by cchung100m**
Hi @srkreddy1238 

This PR is going to add the supplement of deploying a realtime module with lib, graph, and param.
I would appreciate that if you can help to review/manage this PR.

many thanks,

### CODE REVIEW

File: golang/sample/gen_mobilenet_lib.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# specific language governing permissions and limitations
+# under the License.
+
+import tvm
+from tvm import relay
+import tvm.relay.testing
+
+######################################################################
+# Load Neural Network in Relay
+####################################################
```
Comment (srkreddy1238) [inline golang/sample/gen_mobilenet_lib.py]: correct the length up to text length.

File: golang/sample/gen_mobilenet_lib.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+mod, params = relay.testing.mobilenet.get_workload(batch_size=1)
```
Comment (srkreddy1238) [inline golang/sample/gen_mobilenet_lib.py]: Thanks for enhancing the as an end to end example.

How about using an official model than a workload ? 
i.e. Download mobilenet from official tensorflow, import and build. You may refer to tensorflow frontend test cases to download & import an official tensorflow model.

File: golang/sample/gen_mobilenet_lib.py (hunk 1 above)
Comment (cchung100m) [reply golang/sample/gen_mobilenet_lib.py]: Done and thanks for the review.

File: golang/sample/gen_mobilenet_lib.py (hunk 2 above)
Comment (cchung100m) [reply golang/sample/gen_mobilenet_lib.py]: Yes, I use the official model and TFLite frontend for the example, instead.

**Pull request merged**